File: Anul 2/Medii de Programare si Proiectare/Lab/mpp-proiect-csharp-Mothyque/ProiectMPP/ProiectMPP.Common/Domain/Ticket.cs
using System;

namespace ProiectMPP.Common.Domain
{
    [Serializable]
    public class Ticket : Entity<int>
    {
        public int ClientId { get; set; }
        public int MatchId { get; set; }
        public string SeatLocation { get; set; }

        public Ticket(int clientId, int matchId, string seatLocation)
        {
            ClientId = clientId;
            MatchId = matchId;
            SeatLocation = seatLocation;
        }

        public Ticket() { }

        public override string ToString()
        {
            return $"Ticket{{clientId={ClientId}, matchId={MatchId}, seatLocation='{SeatLocation}'}}";
        }
    }
}

File: Anul 2/Medii de Programare si Proiectare/Lab/mpp-proiect-java-Mothyque/backend/Common/src/main/java/org/example/domain/Ticket.java
package org.example.domain;

public class Ticket extends Entity<Integer>
{
    private Integer clientId;
    private Integer matchId;
    private String seatLocation;

    public Ticket(Integer clientId, Integer matchId, String seatLocation)
    {
        this.clientId = clientId;
        this.matchId = matchId;
        this.seatLocation = seatLocation;
    }

    public Ticket() {}
    public Integer getClientId() { return clientId; }
    public void setClientId(Integer clientId) { this.clientId = clientId; }
    public Integer getMatchId() { return matchId; }
    public void setMatchId(Integer matchId) { this.matchId = matchId; }
    public String getSeatLocation() { return seatLocation; }
    public void setSeatLocation(String seatLocation) { this.seatLocation = seatLocation; }

    @Override
    public String toString()
    {
        return "Ticket{" +
                "clientId=" + clientId +
                ", matchId=" + matchId +
                ", seatLocation='" + seatLocation + '\'' +
                '}';
    }
}
